src/sidebar/components/menu-item.js

```diff
@@ -9,6 +9,22 @@ import SvgIcon from '../../shared/components/svg-icon';
 import MenuKeyboardNavigation from './menu-keyboard-navigation';
 import Slider from './slider';
 
+/**
+ * @typedef MenuItemProps
+ * @prop {string} [href] - URL of the external link to open when this item is clicked. Either the `href` or an  `onClick` callback should be supplied.
+ * @prop {string} [iconAlt] - Alt text for icon.
+ * @prop {string} [icon] - Name or URL of icon to display. If the value is a URL it is displayed using an `<img>`, if it is a name it is displayed using `SvgIcon`.  If the property is `"blank"` a blank placeholder is displayed in place of an icon. If the property is falsey, no placeholder is displayed. The placeholder is useful to keep menu item labels aligned in a list if some items have icons and others do not.
+ * @prop {boolean} [isDisabled] - Dim the label to indicate that this item is not currently available.  The `onClick` callback will still be invoked when this item is clicked and the submenu, if any, can still be toggled.
```

Comment:
Would you mind wrapping long (> 100) char prop descriptions for readability?

Related to this, I have visual gutters set up in my editor at 80 and 100 chars that I find helpful to make it clear when a line is long enough that it should be wrapped, since Prettier doesn't currently auto-format these lines. I don't know off-hand how to set up something similar in VS Code but I assume there is a way.

<img width="500" alt="Screenshot 2020-07-28 at 09 44 37" src="https://user-images.githubusercontent.com/2458/88641356-fc13c200-d0b6-11ea-84de-c7c0da5a04fe.png">